secretstores/aws/secretmanager/secretmanager.go

```diff
@@ -109,11 +109,11 @@ func (s *smSecretStore) BulkGetSecret(req secretstores.BulkGetSecretRequest) (se
 				// VersionStage: versionStage,
 			})
 			if err != nil {
-				return secretstores.GetSecretResponse{Data: nil}, fmt.Errorf("couldn't get secret: %s", *entry.Name)
+				return secretstores.BulkGetSecretResponse{Data: nil}, fmt.Errorf("couldn't get secret: %s", *entry.Name)
 			}
 
 			if entry.Name != nil && secrets.SecretString != nil {
-				resp.Data[*entry.Name] = *secrets.SecretString
+				resp.Data[*entry.Name] = map[string]string{*entry.Name: *secrets.SecretString}
```

Comment:
It would be great to see a unit test or integration test added for this functionality. The `GetSecret` function already has tests so it should be a small amount of work.